projects/archiver/src/tasks/front/helpers/media.ts

```diff
@@ -44,12 +44,16 @@ export const getImagesFromArticle = (
     const images = elements.map(getImageFromElement)
     console.log('Images are: ' + JSON.stringify(images))
 
+    const cartoonImages = article.cartoonImages || []
+    console.log('Cartoon images are: ' + JSON.stringify(cartoonImages))
```

Comment:
Please remove

Reply:
I assume you meant the log line? I have removed it here: https://github.com/guardian/editions/pull/2337/commits/aaae0fdcb3bc695ef43b185129cb5f07b4ef2ba6. The other images are logged out in the lines above, so this was done to maintain consistency.